index.js

```diff
@@ -184,9 +170,9 @@ module.exports = class ServeDrive extends ReadyResource {
       return
     }
 
-    const { pathname, searchParams } = new URL(req.url, `http://${LOCALHOST}`)
-    const version = searchParams.get('checkout')
-    const id = searchParams.get('drive') // String or null
+    const { pathname, searchParams } = new URL(req.url, 'http://127.0.0.1')
```

Comment:
Just in case, this URL at the end can be anything, it's just to use the `URL` as a helper to get the `pathname` and `searchParams` from `req.url`. So no need for a constant for that

(re-commented here because I commented in the left version before lol)